backend/src/main/java/com/bulletjournal/messaging/util/FunctionUtil.java

```diff
@@ -0,0 +1,41 @@
+package com.bulletjournal.messaging.util;
+
+import com.bulletjournal.clients.UserClient;
+import com.bulletjournal.controller.models.User;
+import com.bulletjournal.repository.UserAliasDaoJpa;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.BiFunction;
+
+public class FunctionUtil {
+
+  private static final String NONE_STRING = "NONE";
+
+  /**
+   * get user avatar
+   */
+  public static BiFunction<UserClient, String, String> getAvatar = ((userClient, username) -> {
+    User user = userClient.getUser(username);
+    if (user.getAvatar() != null) {
+      return user.getAvatar();
+    }
+    return NONE_STRING;
+  });
+
+  /**
+   * get list of given user alias
+   */
+  public static BiFunction<UserAliasDaoJpa, List<String>, List<String>> getAliases = (
```

Comment:
Move this to UserAliasDaoJpa class?

I don't think we need too many util classes

Reply:
ok. will update later.